db/seeds.rb

```diff
@@ -5,3 +5,16 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+users = User.create(
+  [
+    {email: "sample1@sample.com", password: "password", password_confirmation: "password"},
+    {email: "sample2@sample.com", password: "password", password_confirmation: "password"},
+    {email: "sample3@sample.com", password: "password", password_confirmation: "password"},
+  ]
+)
+
```

Comment:
```
contents = [
  "Lorem Ipsum is simply dummy text of the printing and typesetting industry.",
  "Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.",
  "It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged."
]
```

Using `contents.sample`